dags/attitudes_daily.py

```diff
@@ -7,9 +7,9 @@
 
 
 default_args = {
-    "owner": "ssuh@mozilla.com",
+    "owner": "jklukas@mozilla.com",
```

Comment:
I have no idea who would have more context on this, so marking myself as owner.

Reviewers: feel free to drop any knowledge/context here.